2.0/s2i/bin/run

```diff
@@ -5,20 +5,39 @@ function is_puma_installed() {
   grep ' puma ' Gemfile.lock >/dev/null
 }
 
+function is_rerun_installed() {
+  [ ! -f Gemfile.lock ] && return 1
+  grep ' rerun ' Gemfile.lock >/dev/null
+}
+
 set -e
 
 export RACK_ENV=${RACK_ENV:-"production"}
 
+if [[ "$RACK_ENV" == "development" ]]; then
+  if is_rerun_installed; then
+    export RERUN="rerun --"
```

Comment:
you probably also need to add `--no-growl` (which might stop producing the annoying stty messages)